docs/tutorial/single-node.rst

```diff
@@ -89,20 +59,27 @@ yet any storage available.
 Add storage
 -----------
 
-.. warning::
-
-   This step will remove the data found on the target storage disks. Make sure
-   you don't lose data unintentionally.
+Three OSDs will be required to form a minimal Ceph cluster. In a
+production system, typically we would assign one physical block device
+for an OSD. However for this tutorial, we will make use of file backed
+OSDs for simplicity. In this configuration, we won't be using real
+block devices for storage but instead simulate using regular files.
```

Comment:
this second sentence is redundant. either reword into a single sentence or just remove the second sentence